apps/cli/index.js

```diff
@@ -56,9 +57,36 @@ For more information try ${color('--help', 'green')} or ${color(
   );
 }
 
+async function main() {
+  const message = process.argv[2];
+
+  const validOptions = ['-v', '--version', '-h', ' --help'];
+
+  if (typeof message === 'undefined') {
+    return showHelp();
+  }
+
+  if (message.charAt(0) === '-' && !validOptions.includes(message)) {
+    return showErrorMessage(message);
+  }
+  switch (message.toLowerCase()) {
+    case '-v':
+    case '--version':
+      showVersion();
+      break;
+
+    case '-h':
+    case '--help':
+      showHelp();
+      break;
+
+    default:
+      return await open(
+        `https://twitter.com/intent/tweet?text=${message}%0Acc%20%40sseraphini`,
+      );
```

Comment:
replacing `break` by `return` here, so we can catch it in the tests too. it doesn't affect the ux